code/modules/client/preferences.dm

```diff
@@ -1205,10 +1205,16 @@ var/const/MAX_SAVE_SLOTS = 10
 					commander_sidearm = new_co_sidearm
 
 				if("grade_path")
-					var/list/options = list("Command", "Technical")
+					var/list/options = list(SEA_RANKS[1])
+					if(get_job_playtime(owner, JOB_SEA) >= JOB_PLAYTIME_TIER_1)
+						options += list(SEA_RANKS[2]) + list(SEA_RANKS[3])
+					if(get_job_playtime(owner, JOB_SEA) >= JOB_PLAYTIME_TIER_3)
+						options += list(SEA_RANKS[4]) + list(SEA_RANKS[5])
```

Comment:
I am not sure whether using a `#define`d list this way is a good idea. Be it a global variable, sure, but this feels unseemly (and it can be `options += SEA_RANKS[2]` without a `list()` in the middle anyway). This, though, might be better? (Provided I can use `Cut()` without an off-by-one...)
```suggestion
					var/list/options = SEA_RANKS
					if(get_job_playtime(owner, JOB_SEA) < JOB_PLAYTIME_TIER_3)
						options.Cut(4)
					if(get_job_playtime(owner, JOB_SEA) < JOB_PLAYTIME_TIER_1)
						options.Cut(2)
```